docs/changelog.rst

```diff
@@ -0,0 +1,28 @@
+*****************************
+hipBLASLt Changelog
+*****************************
+
+hipBLASLt
+=============
+
```

Comment:
Could you help me the add that hipblasDatatyp_t has been deprecated and now we use hipblasltDatatype_t?

Reply:
Ok